src/trixi.c

```diff
@@ -119,6 +127,12 @@ void trixi_initialize(const char * project_directory, const char * depot_path) {
         print_and_die("null pointer", LOC);
       }
     }
+
+    // Show version info
+    printf("\nLibTrixi %s\n\n", trixi_version());
```

Comment:
```suggestion
    printf("\nlibtrixi %s\n\n", trixi_version());
```
We style it as `libtrixi`, don't we?